onnx/test/reference_evaluator_test.py

```diff
@@ -3758,6 +3758,106 @@ def test_dequantize_linear_int16(self):
         got = ref.run(None, {"X": data})
         assert_allclose(expected, got[0])
 
+    @parameterized.parameterized.expand(
```

Comment:
Maybe we should test the error message triggered when scale and input are wrong flor block quantization (for one dimension, input shape is not a multiple of the scale shape).

Reply:
Added tests for the failing cases in both `test_blocked_quantize_linear` and `test_blocked_dequantize_linear`